frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -61,6 +72,25 @@ export function Surveys(): JSX.Element {
                     { key: SurveysTabs.Archived, label: 'Archived surveys' },
                 ]}
             />
+            {featureFlags[FEATURE_FLAGS.SURVEYS_SITE_APP_DEPRECATION] && (
+                <div className="space-y-2">
+                    <VersionCheckerBanner />
+
+                    {surveysPopupDisabled ? (
+                        <LemonBanner
+                            type="info"
+                            action={{
+                                type: 'secondary',
+                                icon: <IconSettings />,
+                                onClick: () => openSurveysSettingsDialog(),
+                                children: 'Configure',
+                            }}
+                        >
+                            Survey popups are currently disabled for this project.
```

Comment:
I think users who already have surveys enabled will be a bit confused by this. For users who are already using surveys, maybe a more detailed note like 

`We are migrating surveys app service over to posthog-js. Simply configure and enable the toggle for the transition.`

And then for all others, the regular statement works

Reply:
Hmm, in this case, might be better to let them migrate themselves vs. doing it automatically? - if this is getting confusing.

Since now both versions can exist together without issues, so we can deprecate the surveys app, so no new people can use it, and send a message + have an in app banner explaining migration?

Reply:
I'll defer to whatever you think is better here as long as existing users know that we've moved things over and they're not just totally confused when they log in (even if we auto migrate them and they don't have to do anything) 😄

Reply:
Will email them & stick to same view in app here.